Ghidra/Framework/Utility/src/main/java/ghidra/framework/ConfigurationProperties.java

```diff
@@ -0,0 +1,106 @@
+/* ###
+ * IP: GHIDRA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ghidra.framework;
+
+import generic.jar.ResourceFile;
+import ghidra.util.NullOutputStream;
+import ghidra.util.SystemUtilities;
+import ghidra.util.config.PropertiesEnhance;
+
+import java.awt.*;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collection;
+
+import static ghidra.util.config.ColorHexConvert.toColorFromString;
+
+
+public class ConfigurationProperties extends PropertiesEnhance {
```

Comment:
Insonsistency! The name is called `ConfigurationProperties` but the actual implementation only cares about color. A better name might be `ColorProperties'.